openapi/clusters_mgmt/v1/openapi.json

```diff
@@ -14949,6 +14949,14 @@
           }
         }
       },
+      "AutoNodeMode": {
+        "description": "The configured mode of operation for AutoNode in the Cluster.",
+        "type": "string",
+        "enum": [
+          "Disabled",
```

Comment:
Please use lower case values.

In case the mode is "disabled" do we enforce no arn is provided?

Reply:
@tzvatot As a validation on CS when the payload is submitted?

Reply:
yes